packages/core/styles/_spacing.scss

```
@@ -2,45 +2,45 @@
 @use "../jkl/spacing";
 @use "../jkl/screens";
 
-@each $name, $spacing in spacing.$spacing-new {
-    $small-spacing: map.get($spacing, "small-screen");
-    $large-spacing: map.get($spacing, "large-screen");
-
+@each $name, $spacing in spacing.$spacing {
     .jkl-spacing-#{$name} {
         &--all {
-            margin: $small-spacing;
+            margin: $spacing;
         }
 
         @each $position in spacing.$positions {
             &--#{$position} {
-                margin-#{$position}: $small-spacing;
+                margin-#{$position}: $spacing;
             }
         }
     }
+}
+
+@each $from, $to in spacing.$spacing-pairs {
+    .jkl-spacing-#{$from}-#{$to} {
+        &--all {
+            margin: $from;
```

Disse produserer ikke gyldige verdier.

```suggestion
            margin: map.get(spacing.$spacing, $from);
```

---

D'oh!